src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -15,15 +17,29 @@ namespace MetalPerformanceShaders {
 	public enum MPSKernelOptions : ulong {
 		None									= 0,
 		SkipApiValidation						= 1 << 0,
-		MPSKernelOptionsAllowReducedPrecision	= 1 << 1,
+		AllowReducedPrecision = 1 << 1,
+		[iOS (10,0), TV(10,0)]
+		DisableInternalTiling = 1 << 2,
+		[iOS (10,0), TV (10,0)]
+		InsertDebugGroups = 1 << 3,
+		[iOS (11,0), TV (11,0)]
 		Verbose = 1 << 4,
+#if !XAMCORE_4_0
+		MPSKernelOptionsAllowReducedPrecision = AllowReducedPrecision,
```

Comment:
put an `[Obsolete]` on it - so developers who track (and fix) their warnings can have the smoothest transition to XAMCORE_4_0